Octokit.Tests/OctoKit.Tests-NetCore45.csproj

```diff
@@ -184,6 +184,10 @@
       <Project>{c8bc13b6-3fa3-4716-827d-e7706f976fe1}</Project>
       <Name>Octokit-NetCore45</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Octokit\Octokit.csproj">
```

Comment:
Not sure how this crept in, but this project reference shouldn't be needed :fire: :fire: :fire: